apps/web/pages/team/[slug].tsx

```diff
@@ -84,7 +84,7 @@ function TeamPage({ team }: TeamPageProps) {
   return (
     <div>
       <HeadSeo title={teamName} description={teamName} />
-      <div className="dark:bg-darkgray-50 h-screen rounded-md bg-gray-100 px-4 pt-12 pb-12">
+      <main className="dark:bg-darkgray-50 mx-auto max-w-3xl rounded-md bg-gray-100 px-4 pt-12 pb-12">
```

Comment:
main tag is how embed knows which content should be considered for calculating iframe height.

Teams page is not consistent with [user] page.